bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/upgrade.yaml

```diff
@@ -4,7 +4,6 @@
     backup_dir: "/astronomer-backups"
     timestamp: "{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
     upgrade_to_version: "0.23"
-    upgrade_to_version_airflow: "0.18"
```

Comment:
@danielhoherd @rbankston Remind me again, why we chose to comment this line out? And if left out, what airflow chart version is it upgrading to?

Reply:
Sorry for not doing the review in one go.. Different conversations and slack messages are sparking different thoughts in my head.

Reply:
If that line exists they'll force 0.18.x into 0.23.15/0.23.14 when it should be airflow chart 0.19.0

Reply:
Should we have made it 0.19 then? I am worried it will pick up 0.20.x if we leave it blank.

Reply:
The airflow chart version is used if not defined will inherit from https://github.com/astronomer/astronomer/blob/v0.23.14/charts/astronomer/values.yaml#L8 that locations airflow chart version.

Reply:
ok cool!